lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -67,13 +67,22 @@ def send_ace_message(goal):
         'goals_unsubscribe_url': goals_unsubscribe_url,
         'image_url': settings.LMS_ROOT_URL + settings.STATIC_URL,
         'unsubscribe_url': None,  # We don't want to include the default unsubscribe link
+        'omit_unsubscribe_link': True,
     })
 
+    language = get_user_preference(user, LANGUAGE_KEY)
+    image_url = 'http://courses.edx.org/static/images/'
```

Comment:
I assume this url is planned to be updated to not hardcode our lms?

Reply:
my bad i set it correctly above this was just leftover from my local testing, will remove